formulaire-public/src/main/resources/public/ts/directives/public-question-item.ts

```diff
@@ -49,6 +49,14 @@ class Controller implements IViewModel {
                 this.mapChoiceResponseIndex.set(choice, this.responses.all.indexOf(matchingResponses[0]));
             }
         }
+
+        if (this.question.question_type === Types.CURSOR) {
+            this.responses.all[0].answer =
```

Comment:
this.responses.all ne peut pas etre vide ?

Reply:
Il est initialisé dans le controller parent